### ISSUE TITLE
When running Slap under Docker, text highlight follows mouse cursor even if mouse is not clicked.

### ISSUE THREAD

**Issue by askldjd** [slap-editor/slap#265]
Software:
Docker 1.9.1
Ubuntu 14.04 Docker Image
Slap 1.59
Node 5.3.0

Attached is a screenshot of the problem. When I open Slap on a file from within the Docker image, the text are highlighted following my mouse cursor. It is as if Slap is incorrectly assuming that my mouse left button is pushed down.

I am not sure if this is a Docker problem or Slap problem, any advice would be great.

![screenshot from 2016-01-09 19 11 16](https://cloud.githubusercontent.com/assets/3258724/12222318/ddf05d18-b786-11e5-8239-73f6a7e84d33.png)

**Comment by dbkaplun** [comment]
Hey @askldjd,

Thanks for reporting this. What terminal emulator are you using? If it's GNOME Terminal, this may be an upstream issue: VTE, the library GNOME Terminal uses for rendering the terminal itself, is known to have some issues regarding mouse handling.

@chjj, any thoughts?

Similar issue: #159 
Relevant source: https://github.com/slap-editor/editor-widget/blob/master/lib/Editor.js#L611-L655

**Comment by askldjd** [comment]
Yes, I am using Gnome Terminal. If I am not in the Docker instance, I do not see an issue.
